app/components/navbar.py

```diff
@@ -70,7 +70,7 @@ def Navbar(dropdown=False):
                 ]
             )
         ],
-        href="alert_screen",
+        href="screen",
```

Comment:
Not sure about this renaming ?

Reply:
perhaps "alert_minimal" ?

alert_screen sounds to me as if it could fit both the homepage & that screen

Reply:
or simply "minimal" ? (minimal / simple version of the full homepage)